infrastructure/aws/ecs.tf

```diff
@@ -5,6 +5,7 @@ locals {
   environment_variables = {
     "ELASTIC__API_KEY" : var.elastic_api_key,
     "ELASTIC__CLOUD_ID" : var.cloud_id,
+    "ROOT_INDEX": "redbox-date-${terraform.workspace}",
```

Comment:
Typo in `date` -> `data`

Reply:
[eb53e0d](https://github.com/i-dot-ai/redbox-copilot/pull/482/commits/eb53e0dbc6604b3d591eaaf23268b4d84131ee7e)